cosmic-applet-network/i18n/nl/cosmic_applet_network.ftl

```diff
@@ -1,18 +1,20 @@
 network = Netwerk
-airplane-mode = Vliegtuig modus
-airplane-mode-on = Vliegtuig modus staat aan
-turn-off-airplane-mode = Zet aan om Wi-Fi, Bluetooth en Telefonie te activeren.
-wifi = Wi-Fi
-ipv4 = IPv4 Adres
-ipv6 = IPv6 Adres
+airplane-mode = Vliegtuigmodus
+airplane-mode-on = Vliegtuigmodus staat aan
+turn-off-airplane-mode = Schakel uit om wifi, bluetooth en mobile data te activeren.
+wifi = Wifi
+ipv4 = IPv4-adres
+ipv6 = IPv6-adres
 mac = MAC
 megabits-per-second = Mbps
 connected = Verbonden
-connecting = Aan het verbinden
+connecting = Verbinding maken...
 connect = Verbind
 cancel = Annuleer
+settings = Netwerkinstellingen...
 visible-wireless-networks = Zichtbare draadloze netwerken
-enter-password = Voer het wachtwoord of encryptie sleuter in
-router-wps-button = Je kunt ook verbinden door de "WPS" knop op de router in te drukken
-unable-to-connect = Niet in staat te verbinden met netwerk
-check-wifi-connection = Controleer dat het Wi-Fi is verbonden aan het internet en dat het wachtwoord correct is
+enter-password = Voer het wachtwoord of encryptiesleutel in
```

Comment:
line 16 and 17: making it consisted with https://github.com/pop-os/cosmic-settings/pull/845/files